libbeat/kibana/transform.go

```diff
@@ -118,6 +118,14 @@ func transformField(version *common.Version, f common.Field) (common.MapStr, com
 		field["type"] = t
 	}
 
+	if f.Type == "binary" {
+		field["aggregatable"] = false
+		field["analyzed"] = false
+		field["indexed"] = false
+		field["doc_values"] = false
```

Comment:
I think `doc_values` should be set accordingly to the value in `fields.yml`, but defaulting to `false` for binary, as this is a valid parameter for binary types, according to https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html#binary-params.

Reply:
Could you please add tests for this to ensure values are set properly https://github.com/elastic/beats/blob/master/libbeat/kibana/transform_test.go#L188